apidoc/Titanium/UI/ActivityIndicator.yml

```diff
@@ -134,7 +134,7 @@ properties:
     since: "2.1.0"
     type: String
     default: <pre>#fff</pre>
-    platforms: [mobileweb]
+    platforms: [mobileweb,iOS]
```

Comment:
"iOS" is no valid platform and currently produces an error on the master when validating the docs with "grunt validate". Can you change it to `[mobileweb,iphone,ipad]` ?